docs/scaling/README.md

```diff
@@ -0,0 +1,47 @@
+# Autoscaling
+
+There are three main components to the autoscaling system: autoscaler, activator and queue-proxy.
+
+## Queue-proxy
+Whenever a Serverless Service is deployed by the user, the queue-proxy sidecar container is injected with it. This queue proxy does the following:
```

Comment:
```suggestion
Whenever a Serverless Service is deployed by the user, the queue-proxy sidecar container is injected with it. This queue-proxy does the following:
```

Also, we jump right to the Serverless Service without any background... might be worth doing something a bit more general, like "When a user deploys their application as a Knative Service, their pod is injected with a sidecar called `queue-proxy`. This queue-proxy does... "